tests/baselines/reference/esDecorators-classDeclaration-classSuper.7.js

```diff
@@ -19,9 +19,19 @@ function foo(method: any, _context: any): any {
 }
 
 new B();
+
+// https://github.com/microsoft/TypeScript/issues/53448
+class C {
+	public constructor() {
+		this.val;
+	}
+
+	@foo
+	public get val(): number { return 3; }
+}
```

Comment:
Can you add a test where a subclass with a getter calls `super()`?

Reply:
I've verified that the reported bug (#53448) doesn't repro when `super()` is actually present. The issue was due to `addRange(statements, visitNodes(node.body.statements, visitor, isStatement, ...))` being called twice when there was no `super()` statement in the constructor body.

Reply:
> I can, but I think the reported issue may have been too specific as the root cause was unrelated to the decorated getter.

Unrelated in that it wasn't the fact it was a getter. The issue was that the class was not inherited and thus there was no `super()` call. The reason it didn't break for fields/accessors was because the `__runInitializers()` helper isn't injected into the constructor in those cases since it's injected into the first initializer.